quicksilver/Quicksilver
# Improve ⌘V for non-latin keyboards

If the 'v' key doesn't exist on the current keyboard, fall back to using the key that is at the same location on the current keyboard (keycode 9)

Pretty simple really. If `NDKeyboardLayout` can't find the 'v' key, then just use a hard coded keycode which corresponds to the 'v' key on a British/US keyboard

To test:

Before this merge, change your keyboard layout to something like Korean, then attempt to use "Some text" ⇥ "Paste" (you may need to scroll for paste since you can't search for it using a Korean keyboard!)
Note that it won't work. Try again after merging this
